### ISSUE TITLE
Search Page Navigation Bug

### ISSUE THREAD

**Issue by Mrhb787** [Bookshlf-in/Website#265]
### Problem
- When user goes to some page **x** and opens a book to see book details when coming back the user again reaches to first page.
- _refer video to get a better Idea_ : [Link](https://user-images.githubusercontent.com/56078689/194152834-4f2b0ead-f265-4d2b-84f0-3841a9adda01.mp4)

### Solution
- There can be multiple ways to solve this however i would prefer to use `useContext` here.
- In `UserContext` create a new object named `Search` as below : 
```
Search : {
query : "",
tag : "",
totalPages : "",
page : "",
}
```
- Now use `UserContext` whenever `Search Page` is rendered to get current page.

**Comment by najeebkp** [comment]
Hi @Mrhb787 please assign this to me

**Comment by Mrhb787** [comment]
Hi @najeebkp Assigned it to you. 
- Remember to follow contribution guidelines while making a PR

**Comment by Mrhb787** [comment]
Hi @najeebkp You can send the PR again to same thread.

**Comment by Mrhb787** [comment]
Hi @najeebkp are you still working on this ?

**Comment by najeebkp** [comment]
@Mrhb787 . i tried in lots of ways to crack this. but couldn't find a proper solution.

**Comment by Mrhb787** [comment]
No Worries @najeebkp.

**Comment by Mrhb787** [comment]
Fixed in #271 
Closing this Issue Now
